dbms/src/Storages/DeltaMerge/Segment.h

```diff
@@ -135,9 +135,15 @@ class Segment : private boost::noncopyable
 
     void serialize(WriteBatch & wb);
 
+    /// Attach a new ColumnFile into the Segment. The ColumnFile will be added to MemFileSet and flushed to disk later.
+    /// Usually the ColumnFile should not contain block data. To write block data, you can use `writeToCache`.
```

Comment:
`should not contain block data` seems a little ambiguous. Maybe `The ColumnFile's data is already on disk` is better.